index.js

```diff
@@ -0,0 +1,9 @@
+/*!
+ * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
+ */
+
+'use strict';
+
+export {default as BrQTable} from './BrQTable.vue';
+export {default as BrQModal} from './BrQModal.vue';
```

Comment:
I imagine this list will grow so sorting would be good:
```suggestion
export {default as BrQModal} from './BrQModal.vue';
export {default as BrQTable} from './BrQTable.vue';
```